core/src/main/scala/org/apache/spark/ui/env/EnvironmentPage.scala

```diff
@@ -83,6 +84,7 @@ private[ui] class EnvironmentPage(
         <div class="aggregated-hadoopProperties collapsible-table collapsed">
           {hadoopPropertiesTable}
         </div>
+         }}
```

Comment:
Indentation? Currently, this has one more extra space.

Reply:
@dongjoon-hyun